### PR TITLE
For #13926 - MP migration

### PR DESCRIPTION
**Pull request by ekager**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Comment by ekager** [comment]
mpmigration

gif of behavior w empty logins in Fenix. Also tested case where I had already saved a duplicate login in Fenix and it doesn't make a duplicate or crash. 👍

**Comment by ekager** [comment]
# Request for data collection review form

**All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.**

1) **What questions will you answer with this data?**

- How many users with Master Passwords clicked our homescreen tip to migrate their logins into Fenix?
- How many of those users successfully migrated their logins?

2) **Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements?**

- We want to know how many users successfully "rescued" their logins from Fennec using the master password.

3) **What alternative methods did you consider to answer these questions? Why were they not sufficient?**

* N/A (These are baseline metrics)

4) **Can current instrumentation answer these questions?**
* No, this is a one-time event 

5) **List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox [data c](https://wiki.mozilla.org/Firefox/Data_Collection)[ollection ](https://wiki.mozilla.org/Firefox/Data_Collection)[categories](https://wiki.mozilla.org/Firefox/Data_Collection) on the found on the Mozilla wiki.**

* All data is Category 2.

6) **How long will this data be collected?**

Until 03/01/2020

7) **What populations will you measure?**

* All migrated release, beta, and nightly users with telemetry enabled.

8) **Please provide a general description of how you will analyze this data.**

* Glean / Amplitude

9) **Where do you intend to share the results of your analysis?**

* Only on Glean, Amplitude and with mobile teams.

File: app/metrics.yaml [hunk 1]
```diff
@@ -3946,3 +3946,31 @@ progressive_web_app:
...
+      - https://github.com/mozilla-mobile/fenix/pull/14468#issuecomment-684114534
+    data_sensitivity:
+      - interaction
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2021-03-01"
+  migration:
+    type: event
+    description: |
+      Logins were successfully migrated using a master password.
```
Comment (liuche) [inline app/metrics.yaml:3967]: Do we care about:
- number of people who clicked the banner and/or attempted to migrate
- failures to migrate (such as never entering the correct password, or some other failure?)

Comment (ekager) [reply app/metrics.yaml:3967]: cc @vesta0 for decision. 
Users who clicked the banner is the "displayed" event. 
Users who successfully migrated is the "migration" event.
Failures/dismissals are not currently separated.
